### DISCUSSION TITLE
Starter Prompts with if/else function

### DISCUSSION THREAD

**Discussion by amansoniamazatic** [FlowiseAI/Flowise#1641] [Ideas]
Flowise currently offers the option to include starter prompts. However, I am facing an issue where I am unable to use starter prompts directly within an if/else condition. To work around this, I have to first employ an LLM chain, which I find inconvenient. I suggest adding functionality that allows for the direct use of if/else conditions from the chat input as an input variable. This way, decisions can be made based on the text, determining whether it should be passed to the LLM or not, all within the if/else condition. I believe this approach will save both time and money. Please consider implementing this update in Flowise.

**Comment by HenryHengZJ** [comment]
We are working on a feature called workflow/pipeline which allows you to do sequential if else condition like this. Then you can connect a chatflow depending on the if else condition.
For now, you will have to modify the source code to make the changes.
